### PR TITLE
Setup Travis build.

### PR DESCRIPTION
**Pull request by b4hand**
Also, fix to work with Ruby 2.1 instead of 1.9.

### CODE REVIEW

**Comment by tony-bye** [comment]
and removed `.rvmrc` nice.

File: spec/uri_parser_spec.rb [hunk 1]
```diff
@@ -21,18 +21,18 @@ def self.describe_parsed(url, &block)
     its(:host) { should == 'example.com' }
     its(:path) { should == '/foo/bar' }
     its(:query) { should == 'a=b&c=d' }
-    its(:valid?) { should be_true }
+    its(:valid?) { should be_truthy }
```
Comment (tony-bye) [inline spec/uri_parser_spec.rb]: Why this change?

Comment (b4hand) [reply spec/uri_parser_spec.rb]: Required change due to updated rspec version. The updated rspec version was required to work with Ruby 2.
